src/components/ApyCalculatorModal/index.tsx

```diff
@@ -78,19 +87,36 @@ const ApyCalculatorModal: React.FC<ApyCalculatorModalProps> = ({
 
   return (
     <Modal title={t('ROI')} onDismiss={onDismiss}>
+      {isFarm && (
+        <Flex mb="24px" justifyContent="space-between">
+          <Text small color="textSubtle">
+            {t('APR (incl. LP rewards)')}
+          </Text>
+          <Text small>{apr.toFixed(roundingDecimals)}%</Text>
+        </Flex>
+      )}
       <Grid>
         <GridHeaderItem>
           <Text fontSize="12px" bold color="textSubtle" textTransform="uppercase" mb="12px">
             {t('Timeframe')}
           </Text>
         </GridHeaderItem>
         <GridHeaderItem>
-          <Text fontSize="12px" bold color="textSubtle" textTransform="uppercase" mr="12px" ml="12px" mb="12px">
+          <Text
+            textAlign="right"
+            fontSize="12px"
+            bold
+            color="textSubtle"
+            textTransform="uppercase"
+            mr="12px"
```

Comment:
This nasty stuff was almost definitely added by me originally, but it could be simplified to `m="0 12px 12px 12px"`

Reply:
Bugs actually made a brand-new calculator modal, so the whole thing will soon need to be rewritten anyway :)

Reply:
Ah nice, all good then 👍